makers/staff_engineer.md

```diff
@@ -1,47 +1,78 @@
-* Monitor performance of engineers on team, and partner with engineering managers on mentoring and growth
-* Look for ways to reduce distractions, interruptions, and unplanned work for themselves and others 
-* Challenge their team to demonstrate greater sense of urgency
-* Model bringing solutions to problems (not surfacing problems alone) and teach others on their team how to do the same
-* Be active in the technology community, positively contribute to Mariana Tek’s reputation, and find great candidates to grow the Mariana Tek team
-* Develop and constantly refine technical and team fit assessments (coding exercises, interview questions, etc) for interviews
-* Ensure code/design/architecture reviews happen on teams and that work follows established best practices and standards
-* Coordinate evaluation of tools and frameworks for the engineering team
-* Take responsibility for actions of their team, good and bad
+<!-- TODO: Add summary -->
+
+## Qualifications
+
+- Highly knowledgeable about our product, codebase, and tech stack.
+- Consistently delivers high-quality projects with cross-team impact.
+- Steps up to take leadership of a project or team when needed.
+- Respected by engineers across the company as a resource for technical knowledge, leadership, and mentoring.
+- Trusted by company leaders to contribute strong technical guidance to collaborative decisions.
+- Demonstrated ability to take a nebulous project, perform additional research, architect a solution, break the work into subtasks, and coordinate delivery quickly and efficiently among multiple engineers.
+- Demonstrated ability to diagnose, fix, and coordinate deployment of bug fixes -- and to do this extremely quickly in emergency situations.
+## Responsibilities & Expectations
+
+<!-- TODO: Add summary -->
+
+### Technical Skills
+
+- Take the lead in designing and implementing solutions to complex, unscoped problems. 
+- Design, prioritize, and execute large projects based on strategic business objectives.
+- Work to improve the performance, reliability, and scalability of our applications.
+- Provide high-level technical guidance to the team, anticipating and planning for technical problems.
+- Accurately estimate delivery timelines for shared work, and deliver on those estimates.
+- Facilitate code reviews to ensure the team's work adheres to established standards.
+- Identify and advocate for addressing areas of critical need based on the company's growth plans.
+- Evaluate new technologies and tools for the engineering team to adopt.
+
+Example behaviors:
+- Worked with the PM to break a large feature initiative into smaller tickets before grooming
+- Planned and executed a major version upgrade of an important package
+- Successfully led the team-wide adoption of a new linting tool
+- Flagged a feature that posed a performance risk for a large new client and outlined a plan to refactor it
+
+---
+### Business Knowledge
```

Comment:
I think I want more here - can I suggest that (in my experience) that stating our responsibility to thoroughly understand the risk vs value tradeoff of a product effort is essential at this level.

Maybe something like 'works with Product teams to understand intersection of engineering effort and product initiative'?

Reply:
Good suggestion, will do!

Reply:
Is there any value in using language that highlights a difference between _Industry_ and _Business_ knowledge, or simply a call out that one or both is more important at certain levels or on certain career tracks.

As an example, perhaps there's a large focus on _Industry knowledge_ at the SE 1 & 2 levels, but expectations increase in the _Business knowledge_ as folks approach the Staff/ME/Principle level/Director?

Reply:
Ooh - I like this idea @thomasmatecki ! That phrasing is a nice distinction 💯

Reply:
That's a really good note, @thomasmatecki! I will make some changes.